core/src/main/scala/scalax/collection/hyperedges/ordered/HyperEdge.scala

```diff
@@ -8,7 +8,7 @@ import scalax.collection.generic.{
 /** Undirected hyperedge with ends having sequence semantic with respect to equality.
   */
 @SerialVersionUID(-52)
-final case class HyperEdge[+N] private (override val ends: Several[N])
+final case class HyperEdge[+N](override val ends: Several[N])
```

Comment:
it's not private any more because of the synthesised `copy` method, which doesn't follow the constructor encapsulation in Scala 2, but follows in Scala 3 and it breaks compatibility